examples/aws-text-benchmarks/benchmark_deepsparse.py

```diff
@@ -0,0 +1,66 @@
+from deepsparse import Pipeline, Context
+import deepsparse.transformers
+from datasets import load_dataset
+from transformers import AutoTokenizer
+from tqdm import tqdm
+import numpy as np
+import time, os
+
+os.environ["NM_BIND_THREADS_TO_CORES"] = "1"
+INPUT_COL = "text"
+dataset = load_dataset("ag_news", split="train[:3000]")
+batch_size = 64
+buckets = [64, 128, 256]
+model_path = "./sparse-model/deployment/"
```

Comment:
nit: better to use `os.path.join` for file paths for compatibility between systems